site/README.md

```diff
@@ -0,0 +1,9 @@
+# About
+Source code for the https://devswag.io website.
+
+# Running locally
+1. Clone this repository by running `git clone https://github.com/swapagarwal/swag-for-dev.git`
```

Comment:
I'd rater use codeblocks for commands, easier to copy/paste, but that's personal

Reply:
I added a code block for the `clone` command. Should I put the other commands in blocks also?

Reply:
I'd do it for the npm install too, it's up to you